Automattic/wp-calypso
# Import flow: Get rid of obsolete capture-retired screen and feature flag

## Proposed Changes

* Got rid of obsolete `capture-retired` screen
* Removed unused `onboarding/import-light-url-screen` feature flag

## Testing Instructions

* Start any import flow, for example: `/setup/import-focused?siteSlug={SITE_SLUG}`
* Check if URL capture works as expected

## Pre-merge Checklist
